src/script/abspath.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+#!/bin/sh
+if [ `uname` = Darwin ]; then
+    find `cd $1; pwd` -name $2 -print
```

Comment:
I wrote in this way in order to make it work on Mac.
On Mac, 'uname -o' does not work. != operator in [ does not work either.